kuma/users/views.py

```diff
@@ -179,74 +183,143 @@ def ban_user_and_cleanup_summary(request, user_id):
             submitted_to_akismet.append(revision)
         else:
             not_submitted_to_akismet.append(revision)
+
+        # If there is a current revision and the revision is not in the spam list,
+        # to be reverted, do not revert any revisions
+        if revision.document.current_revision not in revisions_to_mark_as_spam_and_revert:
+            pass
+        else:
+            # If this is a new revision on an existing document, revert it
+            if revision.previous:
+                reverted = revert_document(request=request,
+                                           revision_id=revision.previous.id)
+                if reverted:
+                    revisions_reverted_list.append(revision)
+                else:
+                    # If the revert was unsuccessful, include this in the follow-up list
+                    revisions_not_reverted_list.append(revision)
+
+            # If this is a new document/translation, delete it
+            else:
+                deleted = delete_document(request=request,
+                                          document=revision.document)
+                if deleted:
+                    revisions_deleted_list.append(revision)
+                else:
+                    # If the delete was unsuccessful, include this in the follow-up list
+                    revisions_not_deleted_list.append(revision)
```

Comment:
Also missing coverage here.  Mocking ``delete_document`` would work, if it is possible.